src/Summary/SummaryForm.js

```diff
@@ -70,7 +70,17 @@ class SummaryForm extends React.Component {
         <Col xs={12} md={6}>
           <Field label={<FormattedMessage id="ui-organizations.summary.code" />} name="code" id="code" validate={[Required]} component={TextField} fullWidth required />
           <Field label={<FormattedMessage id="ui-organizations.summary.accountingCode" />} name="erpCode" id="erpCode" component={TextField} fullWidth />
-          <Field label={<FormattedMessage id="ui-organizations.summary.vendorStatus" />} name="status" id="status" validate={[Required]} component={Select} fullWidth dataOptions={statusOptions} required />
+          <Field
+            label={<FormattedMessage id="ui-organizations.summary.vendorStatus" />}
+            name="status"
+            id="status"
+            component={Select}
+            placeholder=" "
```

Comment:
you've added `placeholder` here, but it contradicts with option list above, in `statusOptions` there is `{ label: 'Select Status', value: '' },`

Reply:
fixed